Makefile

```diff
@@ -7,12 +7,6 @@
 include .env
 export
 
-# Set these in the environment to override them. This is helpful for
-# development if you have file ownership problems because the user
-# in the container doesn't match the user on your host.
-USE_UID ?= 10001
-USE_GID ?= 10001
```

Comment:
note that these variables still work, but they have moved to docker-compose.yml for consistency with https://github.com/mozilla-services/tecken/pull/2861/commits/d2671d9e30b4420b8b6edf4ff85f6aa21448ea7d